src/wasm-builder.h

```diff
@@ -27,6 +27,18 @@ class Builder {
 public:
   Builder(AllocatingModule& wasm) : allocator(wasm.allocator) {}
 
+  Function* makeFunction(Name name,
```

Comment:
I also thought about making these functions that take vectors be
```template <typename T>
Function* makeFunction(
...
T&& locals) {
```
Which has the advantage that they will take either an lvalue or an rvalue, so if you want to use the vector that initializes the function after the call you can. That has the downsides that it's less readable (not obvious that you need to pass a `std::vector<NameType>` or something convertible to that) and it means you can't use braced initialization syntax on what you pass without an explicit cast (or a temp variable), which is more verbose and ugly. But if we find ourselves wanting that we can reconsider.